spec.md

```diff
@@ -1398,6 +1509,16 @@ can be mounted on all app instances simultaneously.
 
 \* Fields with an asterisk are REQUIRED.
 
+##### Endpoint Object
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| host* | string | The host name or IP. |
+| port* | string | The port. |
+| proto | string | The protocol. Valid values are `tcp` and `udp`. The default value is `tcp`. |
```

Comment:
Why not require that the proto field be filled with a protocol as mapped in the IANA protocol registry ( https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml , ``/etc/protocols`` )